src/.vuepress/theme/client/components/global/OpenApiRequestParam.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <td>
+    <template v-if="parameter.name">
+      <strong>{{ parameter.name }}</strong>
+      <span
+        v-if="parameter.required"
+        class="text-red-500"
+        title="Required">
+        *
+      </span>
+      <span
+        v-for="flag in flags"
+        :key="flag"
+        >{{ flag }}</span
+      >
+    </template>
+
+    <OpenApiSchema
+      v-if="parameter.schema"
+      :schema="parameter.schema" />
+
+    <Markdown
+      v-if="
+        parameter.description &&
+        !(parameter.schema && 'description' in parameter.schema && parameter.schema.description?.length)
```

Comment:
dit verhoogt niet de leesbaarheid. ik zit bijna te denken dat hier een bepaalde rangorde ook nog inzit qua beschikbaarheid van de properties.
Kan dit weggezet worden in een functie?